projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js

```diff
@@ -17,6 +19,11 @@ import { usePreview } from '../../../../hooks/use-preview';
 import addTokenIntoIframeSource from '../../../../utils/add-token-iframe-source';
 import style from './style.scss';
 
+const VIDEO_PREVIEW_ATTEMPTS_LIMIT = 10;
+
+// The preview is ready when it has a height property
```

Comment:
I considered moving this down to the `usePreview` hook but the web player doesn't check this explicitly. The polling end [conditions are a bit more complicated ](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx#L235). I'm not sure if the preview behavior is different on web but from my observations we need the preview to have an explicit height property before we can let the `Sandbox` component scale properly.